src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+
+  const robot = {
+    name: `${name}`,
+    version: version,
+    wheels: wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    goRight(value = 1) {
+      if (value >= 0) {
+        this.coords.x = this.coords.x + value;
```

Comment:
```suggestion
        this.coords.x += value;
```
всюди можна змінити оператор на компактніший)